win-linux/langs/be.ts

```diff
@@ -222,47 +222,47 @@
     <message>
         <location filename="../src/cfiledialog.cpp" line="230"/>
         <source>EPUB File (*.epub)</source>
-        <translation>Электронная публікацыя (*.epub)</translation>
+        <translation>Файл EPUB (*.epub)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="231"/>
         <source>FB2 File (*.fb2)</source>
-        <translation>Файл FictionBook (*.fb2)</translation>
+        <translation>Файл FB2 (*.fb2)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="232"/>
         <source>MOBI File (*.mobi)</source>
-        <translation>Электронная кніга mobipocket (*.mobi)</translation>
+        <translation>Файл MOBI (*.mobi)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="233"/>
         <source>OFORM Document (*.oform)</source>
-        <translation>Запаўняльная анлайн-форма ONLYOFFICE (*.oform)</translation>
+        <translation>Дакумент OFORM (*.oform)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="234"/>
         <source>DOCXF Document (*.docxf)</source>
-        <translation>Шаблон формы ONLYOFFICE (*.docxf)</translation>
+        <translation>Дакумент DOCXF (*.docxf)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="236"/>
         <source>PPTX File (*.pptx)</source>
-        <translation>Прэзентацыя Office Open XML (*.pptx)</translation>
+        <translation>Файл PPTX (*.pptx)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="237"/>
         <source>PPT File (*.ppt)</source>
-        <translation>Прэзентацыя PowerPoint 97-2003 (*.ppt)</translation>
+        <translation>Файл PPT (*.ppt)</translation>
     </message>
     <message>
         <location filename="../src/cfiledialog.cpp" line="238"/>
         <source>Presentation template (*.potx)</source>
-        <translation>Шаблон прэзентацыі PowerPoint Open XML (*.potx)</translation>
+        <translation>Шаблон прэзентацыі (*.potx)</translation>
```

Comment:
correct source for translation: PowerPoint Open XML Presentation Template (*.potx)